### DISCUSSION TITLE
Welcome to zotero-mdnotes Discussions!

### DISCUSSION THREAD

**Discussion by argenos** [argenos/zotero-mdnotes#57] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jdinning** [comment]
I currently use Zotfile to extract annotations from PDFs and then use mdnotes to create the markdown. However, in the new Zotero 5.097 beta (which is the beta for version 6) I can use the Zotero PDF Reader to create all the annotations and they get saved nicely in a notes file that doesn't require Zotfile to do the extraction.  Also, those note files have the links to the location in the PDF via the Zotero "Show on page" links.  When I try and copy those notes out of Zotero or use mdnotes to generate a markdown file from the annotations created from Zotero PDF Reader, they dont end up well formatted and more importantly, the links back to the PDF are lost.  Do you have any plans to support the new capability Zotero will be adding to their V6 release?

**Reply by argenos** [reply]
I do have plans to support this once it's out of the beta. For now, you can try the following (taken from [here](https://www.zotero.org/support/pdf_reader_preview):

> Annotations are stored in the Zotero database, not in the PDF file, which allows for much more advanced functionality as well as fast syncing. The File menu contains a new option to export PDFs, which will convert any Zotero annotations into standard annotations in the exported file. (An upcoming version will let you omit annotations, but in the meantime you can drag files to your filesystem to copy just the file.) Similarly, when exporting metadata from your library, there's a new “Include Annotations” option under “Export Files”. We plan to support other ways to export annotations in future updates. Your annotations will never be locked in Zotero.

The biggest problem with the changes will be whether the way to access these new annotations is documented somewhere in the [API](https://www.zotero.org/support/dev/client_coding/javascript_api) (not yet), or if I have to dig through their code to get them out. It might be that later on they support the ability to export them themselves.
